kong/clustering/control_plane.lua

```diff
@@ -115,8 +116,10 @@ function _M:export_deflated_reconfigure_payload()
 
   local config_hash, hashes = calculate_config_hash(config_table)
 
+  local current_transaction_id = global.get_current_transaction_id()
   local payload = {
     type = "reconfigure",
+    current_transaction_id = current_transaction_id,
```

Comment:
This, and the call to `get_current_transaction_id` should be guarded behind `log_level = debug` as well.